src/app/asset-page/artstor-viewer/artstor-viewer.component.ts

```diff
@@ -545,6 +545,7 @@ export class ArtstorViewerComponent implements OnInit, OnDestroy, AfterViewInit
             this.osdViewer.viewport.fitBounds(bounds, true)
         } else {
             this.osdViewer.viewport.fitVertically(true)
+            this.osdViewer.viewport.zoomTo(this.osdViewer.viewport.getMinZoom())
```

Comment:
@rehanabbasi `getMinZoom` works great here, but we should only call this in `onInit`, or even in an `afterViewInit` call. This is because each time the viewport size changes, `refreshZoomedView` is called and it resets the image back to it's full zoomed out view.